packages/react-internal/src/components/Image/Image.types.ts

```diff
@@ -73,6 +73,12 @@ export interface IImageProps extends React.ImgHTMLAttributes<HTMLImageElement>,
    * aspect ratio for the image.
    */
   coverStyle?: ImageCoverStyle;
+
+  /**
+   * If true, use browser-level image lazy-loading.
+   * @defaultvalue false;
+   */
+  lazyLoad?: boolean;
```

Comment:
```suggestion
   * If true, use browser-level image lazy-loading.
   * @defaultvalue false;
   */
  loading?: 'auto' | 'lazy' | 'eager';
```

IMO we should keep components closer to native, I suggest to add `loading` prop as it is. It should also default to `auto`:

> auto: Default lazy-loading behavior of the browser, which is the same as not including the attribute.

Reply:
I agree that components should be closer to native. Here's what led me to the decision to abstract the loading prop:

 `auto` is actually just Chromium's own add-on
> Caution: Although available in Chromium, the auto value is not mentioned in the specification. Since it may be subject to change, we recommend not to use it until it gets included. [source](https://web.dev/browser-level-image-lazy-loading/)

The default value is `eager` as mentioned in MDN Web Docs
> eager: Loads the image immediately, regardless of whether or not the image is currently within the visible viewport (this is the default value). [source](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attr-loading)

There being only two options (excluding Chromium's `auto` add-on which is not in the specification) and one being the default, I think it's more beneficial to abstract it and use a boolean instead.

Let me know how I should proceed.

Reply:
@kvcvc - I think what we should do is to simply allow user to pass `loading` prop onto `<img />`, as suggested above. whether what the value should be is completely up to user to decide. as for default value, default value for `loading` is `undefined`. So if user didn't specifically pass `loading` prop, we don't assign `loading` to `img`

Reply:
@xugao OK - refer to latest commit.